2-mandatory/3-financial-times.js

```diff
@@ -5,7 +5,19 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+    let newArray = []
+    let i;
+
+    for (let i = 0, j = 0; allArticleTitles[i] != undefined; i++)
+    {
+        if (allArticleTitles[i].length <= 65)
```

Comment:
Good. The problem stated "65 characters or less" and you made sure to include <=. Well done